examples/azureopenai-private-endpoints/main.tf

```diff
@@ -23,6 +23,38 @@ module "openai" {
       scale_type    = "Standard"
     }
   }
+  diagnostic_settings = {
+    "aoai-svc-diag" = {
+      name                           = "aoai-svc-diag"
+      log_analytics_workspace_id     = azurerm_log_analytics_workspace.this.id
+      log_analytics_destination_type = "AzureDiagnostics"
+
+      audit_log_retention_policy = {
+        enabled = true
+        days    = 30
+      }
+      request_response_log_retention_policy = {
+        enabled = false
+        days    = 30
+      }
+      trace_log_retention_policy = {
+        enabled = false
+        days    = 30
+      }
+      metric_retention_policy = {
+        enabled = false
+        days    = 30
+      }
+    }
+  }
+  network_acls = [{
+    default_action = "Deny",
+    ip_rules       = ["00.00.00.00"]
```

Comment:
@MChorfa this is failing the e2e test:

```
TestExamples/azureopenai-private-endpoints 2023-10-26T08:00:11Z command.go:185: │ Resource Group Name: "openai-private-endpoint"
TestExamples/azureopenai-private-endpoints 2023-10-26T08:00:11Z command.go:185: │ Account Name: "azure-openai-751912"): unexpected status 400 with error: BadRequest: Invalid IP address or range 00.00.00.00.
```

Have you tested the example ? does it work for you ? did you have a different value ?